CONTRIBUTING.md

```diff
@@ -1,3 +1,107 @@
+
+
+## Types of Contributions
+
+Just as contributors come in many forms (student, faculty, interested
+programmer), so too do contributions. Some of the most valuable contributions
+include reporting issues and requesting new features. Improvements to the
+project can also take the form of minor or major fixes to existing content,
+contributions to process documentation, or creating brand new nodes. 
+
+### Issues
+
+When you notice something wrong or a desirable feature that is missing, you can
+help us by reporting it as an [issue](https://github.com/necx-org/nodes/issues) in this
+repository. Perhaps you've noticed bad grammar, formatting, or malformed
+content. Alternatively, perhaps you have had an idea for a nice potential
+whiz-bang feature and you'd like to suggest it. Either way, the process is the
+same:
+
+1. Search the issues to determine whether an existing issue addresses your
+   bug/feature already. If one appears, comment within that issue to add to the
+   conversation. 
+2. If no existing issue matches your needs, create one.
+3. Include as many details as possible to describe the bug or desired feature. 
+
+### Meta Changes
+
+The NECX project relies on many processes and the documentation explaining
+them. Contribution to documentation and procedures are welcome.  Lively review
+and discussion of such changes can be expected, and consensus will enable
+community adoption. 
+
+In any case, the process will follow the [contributor workflow.](#contributor-workflow)
+
+### Making a Change to a Node
+
+Perhaps an existing node could use improvement, corrections, or new features.
+Ideally, the original author of the node should be available and open for
+discussion (in an issue or pull request) about how best to integrate your
+changes into the node, but their permission is not necessary. Your changes will
+undergo community review in the [contributor workflow](#contributor-workflow)
+process. Accordingly, the node author is free to engage in the review of your
+changes, but may choose not to -- and that's ok!  
+
+In any case, the process will follow the [contributor workflow.](#contributor-workflow)
+
+### Creating a Node 
+
+A major effort in this project is the development of nuclear engineering
+curriculum content in the form of 'nodes'. A node template appears in the
+repository and the review criteria for node contributions can be found
+in the [node review criteria document](./NODE-CRITERIA.md). 
+
+
+Start by copying the template node directory and work from there. Beyond that,
+the process will follow the [contributor workflow.](#contributor-workflow)
+
+## Contributor Workflow
+
+- The terminology we use is based on the [Integrator Workflow](http://en.wikipedia.org/wiki/Integrator_workflow).
+- The first step is to create your own [fork](https://help.github.com/articles/fork-a-repo/) of the project.
+- Use a branching workflow similar to the one described in the [pro-git book](http://progit.org/book/ch3-4.html).
```

Comment:
This link doesn't work for me.

Reply:
Maybe this URL is better: https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows